src/main/java/net/cytonic/cytosis/managers/VanishManager.java

```diff
@@ -21,10 +20,8 @@ public VanishManager() {
      * @param player the player to vanish
      */
     public void enableVanish(Player player) {
-        player.setAutoViewable(false);
-        for (Player onlinePlayer : Cytosis.getOnlinePlayers()) {
-            onlinePlayer.removeViewer(player);
-        }
+        player.updateViewableRule(p -> p.hasPermission("cytosis.vanish.can_see_vanished"));
+        //todo events?
```

Comment:
Just a qol feature, would it be possible to show the player as invisible but with the team option to see friendly invisibles?

![image](https://github.com/user-attachments/assets/2d8c5a18-8766-48b3-86f8-2b73c492ad53)
 (Like this cow)